src/spine.sig

```diff
@@ -1,3 +1,10 @@
+(* A spine is an abstraction over the sequences of symbols and variables
+ * that occur in binders. So that [x.y.] portion of [natrec(a; b; x.y.c)]
```

Comment:
Spines are used for *all* the "collection" type things in ABTs, so in addition to binders and valences, also the arguments of operator & metavariable applications. Maybe it would be clearest to just remove the example here?